src/huggingface_hub/fastai_utils.py

```diff
@@ -0,0 +1,324 @@
+        commit_message (:obj:`str`, `optional`):
+            Message to commit while pushing. Will default to :obj:`"add model"`.
+        private (:obj:`bool`, `optional`):
+            Whether or not the repository created should be private (requires a paying subscription).
+        use_auth_token (:obj:`bool` or :obj:`str`, `optional`):
+            The token to use as HTTP bearer authorization for remote files. If :obj:`True`, will use the token
+            generated when running :obj:`transformers-cli login` (stored in :obj:`~/.huggingface`). Will default to
+            :obj:`True`.
+        config (:obj:`dict`, `optional`):
+            Configuration object to be saved alongside the model weights.
+
+    Keyword Parameters:
+        api_endpoint (:obj:`str`, `optional`):
+            The API endpoint to use when pushing the model to the hub.
+        git_user (:obj:`str`, `optional`):
+            Will override the ``git config user.name`` for committing and pushing files to the hub.
+        git_email (:obj:`str`, `optional`):
+            Will override the ``git config user.email`` for committing and pushing files to the hub.
+        pickle_protocol (:obj:`int`, `optional`):
+            Pickle protocol passed to torch.save. Refer to pickle documentation.
+
+    Returns:
+        The url of the commit of your model in the given repository.
+    """
+
+    # Check that fastai and fastcore versions are supported.
+    check_fastai_fastcore_versions()
+
+    # Unpacking **kwargs
+    organization: str = kwargs.get("organization", None)
+    api_endpoint: str = kwargs.get("api_endpoint", None)
+    git_user: str = kwargs.get("git_user", None)
+    git_email: str = kwargs.get("git_email", None)
+    pickle_protocol: int = kwargs.get("pickle_protocol", 2)
+
+    # Split `repo_id` into organization/user and repo
+    temp = repo_id.split("/")
+    organization = temp[0]
+    repo_name = temp[1]
+
+    if isinstance(use_auth_token, bool) and use_auth_token:
+        token = HfFolder.get_token()
+    elif isinstance(use_auth_token, str):
+        token = use_auth_token
+    else:
+        token = None
+
+    if token is None:
+        raise ValueError(
+            "You must login to the Hugging Face Hub. There are two options:"
+            "(1) Type `huggingface-cli login` in your terminal and enter your token."
+            "(2) Enter your token in the `use_auth_token` argument."
+            "Your token is available in the Settings of your Hugging Face account."
+        )
+
+    # If the repo does not exist then create it using `HfApi()`.
+    repo_url = None
+    if not os.path.exists(repo_id):
+        repo_url = HfApi(endpoint=api_endpoint).create_repo(
+            repo_name,
+            token=token,
+            organization=organization,
+            private=private,
+            repo_type=None,
+            exist_ok=True,
+        )
```

Comment:
Since we are always referring to a remote repo, it can be 
```suggestion
    repo_url = HfApi(endpoint=api_endpoint).create_repo(
            repo_name,
            token=token,
            organization=organization,
            private=private,
            repo_type=None,
            exist_ok=True,
        )
```